resources/js/Pages/Company/HR/AskMeAnything/Answered.vue

```diff
@@ -0,0 +1,131 @@
+<style lang="scss" scoped>
+.question-item:first-child:hover {
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+}
+
+.question-item:last-child {
+  border-bottom: 0;
+
+  &:hover {
+    border-bottom-left-radius: 10px;
+    border-bottom-right-radius: 10px;
+  }
+}
+</style>
+
+<template>
+  <layout :notifications="notifications">
+    <div class="ph2 ph0-ns">
+      <breadcrumb :with-box="true"
+                  :previous-url="'/' + $page.props.auth.company.id + '/company/hr/ask-me-anything'"
+                  :previous="$t('app.breadcrumb_ama_list')"
+                  :has-more="false"
+      >
+        {{ $t('app.breadcrumb_ama_show') }}
+      </breadcrumb>
+
+      <!-- BODY -->
+      <div class="mw7 center br3 mb5 relative z-1">
+        <!-- title -->
+        <div class="bg-white box pa3 center mb4">
+          <h2 class="tc normal mb4 lh-copy">
+            {{ $t('company.hr_ama_show_title', {date: data.happened_at}) }}
+
+            <help :url="$page.props.help_links.wiki" :top="'1px'" />
+          </h2>
+
+          <!-- session theme -->
+          <p v-if="data.theme" class="mb3 tc mv0">{{ data.theme }}</p>
+        </div>
+
+        <!-- tabs -->
+        <div class="cf mw7 center br3 mb5 tc">
+          <div class="cf dib btn-group">
+            <inertia-link :href="data.url.unanswered_tab" class="f6 fl ph3 pv2 dib pointer no-underline" :class="{'selected':(tab == 'unanswered')}">
```

Comment:
You can remove quotes and parenthensis, is easier to read
```suggestion
            <inertia-link :href="data.url.unanswered_tab" class="f6 fl ph3 pv2 dib pointer no-underline" :class="{ selected: tab === 'unanswered' }">
```

Reply:
I didn't know that!!!